logprep/processor/pre_detector/ip_alerter.py

```diff
@@ -38,7 +39,8 @@ def _init_alert_ip_list(self, alert_ip_lists: List):
         for alert_ip_list in alert_ip_lists:
             if alert_ip_list and isfile(alert_ip_list):
                 with open(alert_ip_list, "r", encoding="utf8") as alert_ip_list_file:
```

Comment:
Do we still need this `with open(...)` line?

Reply:
sure not

DONE